external/fv3fit/fv3fit/emulation/transforms/transforms.py

```diff
@@ -71,8 +71,37 @@ def backward(self, x: tf.Tensor) -> tf.Tensor:
         return tf.math.exp(x)
 
 
+@dataclasses.dataclass
+class PositiveTransform:
+    """
+    A univariate transformation for::
+
+    y := x
+    x := ReLU(y)
+
+    Attributes:
+        enforce_positive: used only for identifying this transform in the
+            configuration
+    """
+
+    # TODO: Is there a better way to do config instance via dacite w/o uniqueness?
+    enforce_positive: str = "yes_please"
+
+    def __post_init__(self):
+        self.relu = tf.keras.layers.ReLU()
+
+    def forward(self, x: tf.Tensor) -> tf.Tensor:
+        return x
+
+    def backward(self, x: tf.Tensor) -> tf.Tensor:
+        return self.relu(x)
+
+
+UnivariateCompatible = Union[LogTransform, PositiveTransform]
+
+
 class UnivariateTransform(TensorTransform):
-    def __init__(self, source: str, to: str, transform: LogTransform):
+    def __init__(self, source: str, to: str, transform: UnivariateCompatible):
```

Comment:
After deleting `EnforcePositiveValues` you can make this interface a bit more convenient by letting `to` default to `source`:
```suggestion
    def __init__(self, source: str, to: Optional[str] = None, transform: UnivariateCompatible):
        to = to or source
```

Then you can configure the positivity enforcment like this:
```
tensor_transforms:
- {source: air_temperature_after_gscond, transform: {enforce_positive: True}}
```